pkg/kubelet/cm/dra/manager.go

```diff
@@ -67,7 +68,7 @@ func NewManagerImpl(kubeClient clientset.Interface, stateFileDirectory string, n
 // containerResources on success.
 func (m *ManagerImpl) PrepareResources(pod *v1.Pod) error {
 	batches := make(map[string][]*drapb.Claim)
-	claimInfos := make(map[types.UID]*ClaimInfo)
+	resourceClaims := make(map[types.UID]*resourceapi.ResourceClaim)
```

Comment:
Why not just continue to track claimInfos? That way we can remain symmetrical with `UnprepareResources()`

Reply:
`UnprepareResources` tracks claim names. Both changes decrease usage of claimInfo objects outside of cache operations. I thought that it's much more clear if claimInfo only used when cache is locked.

Reply:
Then I'd argue we should drop `get()` altogether and perform all operations through the cache once a claimInfo has been added (including things like `isPrepared()`.

Reply:
Can you elaborate? Are you talking about `claimInfoCache.get`? I'd love to drop it, but how is it related to this code? This code was added here to avoid using not cached claim infos.

Reply:
Because it feels odd to me to call `get()` to pull something called `claimInfo` out of the cache, and then not use that to track the info about the claim across the calls.

Reply:
It looks like it's a matter of taste. claimInfo object is used to check `prepared` field. This saves one lock of cache comparing to your code and I did it to decrease a number of cache locks. However, now I understand what you mean. Will try to do what you've suggested.